README.md

```diff
@@ -111,7 +114,29 @@ const config = {
 };
 ```
 
-### 5. Start development
+### 5. Add internationalization support
+> The plug-in project needs to be integrated into the dtable project for execution, so the plug-in project needs to be packaged into a js file for uploading and loading, and the translation file also needs to be packaged into a js file.
+
+Recommended use [react-intl-universal](https://github.com/alibaba/react-intl-universal) complete plugin internationalization
```

Comment:
We recommend to use [react-intl-universal](https://github.com/alibaba/react-intl-universal) for plugin internationalization.